src/main/java/gov/cms/ab2d/eventlogger/api/EventsListener.java

```diff
@@ -10,16 +11,21 @@
 @Slf4j
 @Service
 public class EventsListener {
-
     private LogManager logManager;
 
     public EventsListener(LogManager logManager) {
         this.logManager = logManager;
     }
 
     @SqsListener(value = "ab2d-events", deletionPolicy = SqsMessageDeletionPolicy.ON_SUCCESS)
-    public void processEvents(LoggableEvent loggableEvent) {
-        log.info("EventsListener: Processing events from SQS: " + loggableEvent);
-        logManager.log(loggableEvent);
+    public void processEvents(SQSMessages sqsMessage) {
+        log.info("EventsListener: Processing events from SQS: " + sqsMessage);
+        switch (sqsMessage.getClass().getSimpleName()) {
+            case "GeneralSQSMessage":
```

Comment:
Might be easier to maintain by using instanceof instead of a String.

Reply:
you would think. But the switch dosen't support classes. So I kept it with string.

Reply:
Makes it more readable then a bunch of if statments